src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,17 +1,24 @@
 // Node modules.
 import React from 'react';
-import moment from 'moment';
 import download from 'downloadjs';
+import moment from 'moment';
 // Relative imports.
 import * as customPropTypes from '../prop-types';
 
-const onDownloadClick = url => event => {
-  // Escape early if we're not on IE.
-  if (!navigator.msSaveBlob) {
-    return;
+// Helper to check if the URL is the same as our current origin.
+const isURLSameOrigin = url => {
+  const matches = url.match(/^(https?:)?\/\/([^/]+)/);
```

Comment:
Needing to do this instead of something like `new URL(url).hostname === window.location.hostname` since IE doesn't support `new URL()`: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL